x-pack/plugins/observability_solution/synthetics/server/synthetics_service/synthetics_service.ts

```diff
@@ -170,6 +174,9 @@ export class SyntheticsService {
             // Perform the work of the task. The return value should fit the TaskResult interface.
             async run() {
               const { state } = taskInstance;
+              service.logger.debug(`Running synthetics monitors sync task.`);
```

Comment:
From what I have seen on the logs, Is there anyway to add logs to the parent task that invokes this 'Synthetics Service - Sync Saved Monitors'? 

Also it would be nice to log the actual task title instead of this one so we can track it easily rathan than having different names.